docs/reference/migration/migrate_8_0/http.asciidoc

```diff
@@ -44,9 +43,6 @@ In these previous versions, if your application required handling `+` as a singl
 to be supported in version 8.
 
 *Impact* +
-Update your workflow and applications to assume `+` in a URL is encoded as
-`%2B`. Discontinue use of the `es.rest.url_plus_as_space` system property.
-Specifying this property in `elasticsearch.yml` will result in an error on
-startup.
+Update your application or workflow to assume `+` in a URL is encoded as `%2B`.
```

Comment:
@original-brownbear Related to #33164: In an 8.0 snapshot, I can still set `-Des.rest.url_plus_as_space=true` in JVM options and not receive an error. I wouldn't expect this based on the above text:

> Note that this behaviour is deprecated and setting this system property to `true` will cease
to be supported in version 8.

Reply:
Thanks for pining @jrodewig looks like we need to indeed take action here for `8.x` and make it so an error at startup happens.